src/domains/idpay-app/04_apim_api_mil.tf

```diff
@@ -23,48 +23,6 @@ module "idpay_api_mil_product" {
   groups = ["developers"]
 }
 
-## IDPAY MIL API ##
-module "idpay_mil" {
-  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//api_management_api?ref=v6.15.2"
-
-  name                = "${var.env_short}-idpay-mil"
-  api_management_name = data.azurerm_api_management.apim_core.name
-  resource_group_name = data.azurerm_resource_group.apim_rg.name
-
-  description  = "IDPAY MIL"
-  display_name = "IDPAY MIL API"
-  path         = "idpay/mil/payment/qr-code/merchant"
-  protocols    = ["https"]
-
-  service_url = "${local.ingress_load_balancer_https}/idpaypayment/idpay/payment/qr-code/merchant"
-
-  content_format = "openapi"
-  content_value  = file("./api/idpay_mil/openapi.mil.yml")
-
-  xml_content = file("./api/base_policy.xml")
-
-  product_ids           = [module.idpay_api_mil_product.product_id]
-  subscription_required = true
-
-  api_operation_policies = [
-    {
-      operation_id = "getMerchantInitiativeList"
```

Comment:
the operations `getMerchantInitiativeList` and `uploadMerchantList` are removed?

Reply:
the operations getMerchantInitiativeList and uploadMerchantList are removed from API MIL generic, they are moved to idpay_mil_merchant